src/webgl-utils/polyfill-get-parameter.js

```diff
@@ -9,23 +9,25 @@ const EXT_disjoint_timer_query_webgl2 = 'EXT_disjoint_timer_query_webgl2';
 const EXT_texture_filter_anisotropic = 'EXT_texture_filter_anisotropic';
 const WEBGL_debug_renderer_info = 'WEBGL_debug_renderer_info';
 
-const GL_FRAGMENT_SHADER_DERIVATIVE_HINT = 0x8B8B;
-const GL_DONT_CARE = 0x1100;
-const GL_GPU_DISJOINT_EXT = 0x8FBB;
-const GL_MAX_TEXTURE_MAX_ANISOTROPY_EXT = 0x84FF;
-const GL_UNMASKED_VENDOR_WEBGL = 0x9245; // vendor string of the graphics driver.
-const GL_UNMASKED_RENDERER_WEBGL = 0x9246; // renderer string of the graphics driver.
+export const GL_FRAGMENT_SHADER_DERIVATIVE_HINT = 0x8B8B;
```

Comment:
These should not be exported. We added these local constants only to enable minification. 

If you export them, you defeat that. 

In other places you typically use GL.<constant>